src/client/components/app/Transactions/DepositTx.tsx

```diff
@@ -234,23 +243,26 @@ export const DepositTx: FC<DepositTxProps> = ({
   };
 
   const approve = async () => {
-    await dispatch(
-      VaultsActions.approveDeposit({ vaultAddress: selectedVault.address, tokenAddress: selectedSellToken.address })
-    );
+    selectedSellToken &&
+      (await dispatch(
+        VaultsActions.approveDeposit({ vaultAddress: selectedVault.address, tokenAddress: selectedSellToken.address })
+      ));
```

Comment:
nit: Early return a bit more readable IMO
```suggestion
    if (!selectedSellToken) return;

    await dispatch(
      VaultsActions.approveDeposit({ vaultAddress: selectedVault.address, tokenAddress: selectedSellToken.address })
    );
```